scripts/generate-google3-docs/resource-reference/_toc.yaml

```diff
@@ -33,6 +33,10 @@ toc:
     path: /config-connector/docs/reference/resource-docs/alloydb/alloydbinstance.md
   - title: "AlloyDBUser"
     path: /config-connector/docs/reference/resource-docs/alloydb/alloydbuser.md
+- title: "App Hub"
```

Comment:
Probably we don't want to block on this, but maybe we introduce this before promotion